src/components/Dropdown/DropdownContentInner.tsx

```diff
@@ -8,9 +8,10 @@ import { Rect, getContentBoxStyle, ContentBoxStyle } from './dropdownHelper'
 type Props = {
   triggerRect: Rect
   children: React.ReactNode
+  className: string
 }
 
-export const DropdownContentInner: FC<Props> = ({ triggerRect, children }) => {
+export const DropdownContentInner: FC<Props> = ({ triggerRect, children, className = '' }) => {
```

Comment:
[MUST] default parameter is not required because `className` is a required type.

Reply:
😳

Reply:
fixed: 2902401